### PR TITLE
Changes for week of Sept 3

### PR DESCRIPTION
**Pull request by duglin**
- Updated message, endpoint and schema specs to latest common attributes
- lots of clean-up/catch-up

### CODE REVIEW

File: tools/dict [hunk 1]
```diff
@@ -114,24 +186,39 @@ unicode
...
+www
 xml
+xmlschema
+xpath
 xreg
 xregbasicmodel
 xregfullmodel
 xregistries
 xregistry
+xregsitry
```
Comment (jskeet) [inline tools/dict:220]: Really?

Comment (jskeet) [reply tools/dict:220]: (These are typos on lines 218 and 293 of the new version of schema/spec.md)

**Review by jskeet — COMMENTED** [review]
Mostly looks fine, other than the "add a typo to dict instead of fixing it" :)

File: tools/dict (hunk 1 above)
Comment (duglin) [reply tools/dict:220]: yea, this is why I need to you remove my stuff :-)
thanks!

**Review by jskeet — APPROVED** [review]
Glad to be able to be mildly useful :)

**Comment by duglin** [comment]
> Glad to be able to be mildly useful :)

More than mildly... I missed your review last week

**Comment by duglin** [comment]
Approved on the 9/5 call

**Pull request merged**